roles/collect_modified_images/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+- name: "Create empty list for images"
+  set_fact:
+    pulled_images: []
+
+- name: "Generate events.json"
+  shell: "{% raw %}oc get events --field-selector reason==Pulling -o go-template='{{range .items}}{{.lastTimestamp}},{{.message}}{{\"\\n\"}}{{end}}' --sort-by={.lastTimestamp}{% endraw %}"
```

Comment:
Should `oc` be hardcoded here, or should we use `oc_bin_path`?

Reply:
Nope, good catch.  I changed that for testing